packages/frontend-api/src/Resources/config/graphql-types/SettingsDecorator.types.yaml

```diff
@@ -24,3 +24,7 @@ SettingsDecorator:
                 type: String!
                 resolve: "@=query('displayTimezoneQuery')"
                 description: "Timezone that is used for displaying time"
+            blogUrl:
```

Comment:
Would not be better to name it as "mainBlogCategoryUrl" or "rootBlogCategoryUrl"?